### PR TITLE
Fixes #3270.

### LINKED ISSUE #3270: Cannot bind to isAttached
This used to work and no longer does.

### PR DESCRIPTION
**Pull request by sorvell**

### CODE REVIEW

**Comment by azakus** [comment]
LGTM

File: test/unit/bind-elements.html [hunk 1]
```diff
@@ -468,3 +468,23 @@
...
+        isAttached: {
+          observer: '_isAttachedChanged'
+        }
+      },
+      _isAttachedChanged: function() {}
+    });
+  </script>
+</dom-module>
+
+<s
```
Comment (samccone) [inline test/unit/bind-elements.html]: intentional?

Comment (azakus) [inline test/unit/bind-elements.html]: Unlikely. I removed it with https://github.com/Polymer/polymer/commit/d7567b7431d07f177d89b0ee1750929df9e94686

Comment (samccone) [inline test/unit/bind-elements.html]: thanks

**Pull request merged**